install/installer/docs/workspace-ssh-access.md

```diff
@@ -0,0 +1,26 @@
+# Installing Gitpod with SSH access to workspaces with the Gitpod Installer
+**_Needed by some desktop IDEs to connect to a workspace._**
+
+> **IMPORTANT:** If you use hostPort or nodePort as the method for exposing ports to the public, This will uses port 22 on your Kubernetes nodes. When enabled, this will prevent login to the cluster via SSH. If you wish to maintain SSH access to your cluster, please configure another SSH port on your nodes.
```

Comment:
Could you explain this from a user perspective? I'm not aware that I changed the exposing port method somewhere but had a problem with my SSH connection after enabling the SSH gateway. I used k3s on a VM and I was not able to connect to the VM via SSH anymore. See also [this internal discussion](https://gitpod.slack.com/archives/C01KGM9BH54/p1645522647464259).

How do I now as a user of the Gitpod Installer whether I'm affected?

Reply:
Ah, ok. What do you think about this?

```suggestion
> **IMPORTANT:** When you use k3s, this will open port 22 on your Kubernetes nodes for accessing the workspaces. This will prevent login to the cluster via SSH. If you wish to maintain SSH access to your cluster, please configure another SSH port on your nodes.
```